pandas/core/frame.py

```diff
@@ -1840,6 +1844,60 @@ def info(self, verbose=None, buf=None, max_cols=None, memory_usage=None,
             - If True, always show counts.
             - If False, never show counts.
 
+        Returns
+        -------
+        None: NoneType
+            This method outputs a summary of a DataFrame and returns None.
+
+        Examples
+        --------
+        >>> int_values = [1, 2, 3, 4, 5]
+        >>> text_values = ['alpha', 'beta', 'gamma', 'delta', 'epsilon']
+        >>> float_values = [0.0, 0.25, 0.5, 0.75, 1.0]
+        >>> df = pd.DataFrame({"int_col": int_values, "text_col": text_values, "float_col": float_values})
+        >>> df
+           int_col text_col  float_col
+        0        1    alpha       0.00
+        1        2     beta       0.25
+        2        3    gamma       0.50
+        3        4    delta       0.75
+        4        5  epsilon       1.00
+
+        >>> df.info(verbose=True)
+        <class 'pandas.core.frame.DataFrame'>
+        RangeIndex: 5 entries, 0 to 4
+        Data columns (total 3 columns):
+        int_col      5 non-null int64
+        text_col     5 non-null object
+        float_col    5 non-null float64
+        dtypes: float64(1), int64(1), object(1)
+        memory usage: 200.0+ bytes
+
+        >>> df.info(verbose=False)
+        <class 'pandas.core.frame.DataFrame'>
+        RangeIndex: 5 entries, 0 to 4
+        Columns: 3 entries, int_col to float_col
+        dtypes: float64(1), int64(1), object(1)
+        memory usage: 200.0+ bytes
+
+        >>> file = open("df_info.txt", "w", encoding="utf-8")
+        >>> df.info(buf=file)
+        >>> file.close()
+
+        >>> df.drop('text_col', axis=1, inplace=True)
```

Comment:
There is no reason to use df.drop in this example.

Reply:
Done!